CarbonChat-Bukkit/build.gradle

```diff
@@ -0,0 +1,103 @@
+plugins {
+  id 'com.github.johnrengelman.shadow' version '6.0.0'
+  id 'net.minecrell.plugin-yml.bukkit' version '0.3.0'
+}
+
+description = 'CarbonChat-Bukkit'
+
+dependencies {
+  compile 'org.spongepowered:configurate-hocon:4.0.0-SNAPSHOT'
+  compile 'net.kyori:adventure-text-serializer-bungeecord:4.0.0-SNAPSHOT'
+  compile 'net.kyori:adventure-platform-bukkit:4.0.0-SNAPSHOT'
+
+  compile project(":CarbonChat-Common")
+
+  // Messaging
+  compile 'io.github.leonardosnt:bungeechannelapi:1.0.0-SNAPSHOT'
+
+  // Commands
+  compile "com.intellectualsites:cloud-paper:1.0-SNAPSHOT"
+
```

Comment:
implementation / api instead of compile

Reply:
yeah, `compile` is gonna go away in gradle 7